Yoast/Yoast-theme-public
# Fields in checkout are no longer all required when using discount code

There's an issue in EDD where the assumption is made that all fields during checkout are required. This includes both the Address 2 line  and VAT number inputs.
This makes no sense, so this PR removes the required attributes after the discount has successfully been applied.

To test this:
1. Add an item to your cart (i.e. my.yoast.dev/checkout/?yst_action_edd=add_to_cart&download_id=105243).
2. Apply a discount code.
3. Notice how the Address 2 and VAT number fields are no longer required when hitting the Continue button.
4. Rejoice!

Fixes https://github.com/Yoast/yoast.com/issues/852
